### ISSUE TITLE
About ElasticDistortion

### ISSUE THREAD

**Issue by RayYoh** [Pointcept/Pointcept#218]
Hi Xiaoyang. Are there any resources that can help understand ElasticDistortion data aug?
I have searched other open-sourced codes like [PointGroup](https://github.com/dvlab-research/PointGroup/blob/a41c2cd22808861f0d4986b2825875c10bb661e5/data/scannetv2_inst.py#L184C13-L186C82), [Oneformer3D](https://github.com/oneformer3d/oneformer3d/blob/d74fa0cc9cda2c967b26cf7df3b99e7aedbf2086/oneformer3d/transforms_3d.py#L39C1-L42C69). All these code first div voxel_size or time voxel_scale, and then do the data augmentation use the same parameters while Pointcept uses different parameters. I am not sure whether this is a strong augmentation for the data and then further influences the final performance.

**Comment by Gofinge** [comment]
Hi, I think Spatio-Temporal Segmentation by Chrischoy is the first introduced ElasticDistortion. And I think the order of ElasticDistortion and GridSampling doesn't matter.

**Comment by RayYoh** [comment]
Yeah, it's true the order doesn't matter. But I have not understood the distortion parameters, I mean granularity and magnitude. Which have different scales. [[6, 40], [20, 160]] vs [[0.2, 0.4], [0.8, 1.6]]. I'm not sure whether I ignore some operation before ElasticDistortion.

**Comment by Gofinge** [comment]
Oh, actually, I also don't fully understand the meaning of these parameters. I just transfer them from Chris Choy's repo.

**Comment by RayYoh** [comment]
Alright, thanks for your reply.
